frontend/src/components/QuizViewer.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import Question from '../components/Question';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+
+class QuizViewer extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
eventually may have to save :floppy_disk:  the state of previously completed questions for consistency